templates/oe_contact/oe-contact--full.html.twig

```diff
@@ -10,7 +10,11 @@
   <div class="ecl-col-12">
     <h3 class="ecl-u-type-heading-3 ecl-u-mt-none">{{ content.name }}</h3>
   </div>
-  <div class="ecl-col-12 ecl-col-md-6">
+  {% if entity.oe_image is not empty %}
+    <div class="ecl-col-12 ecl-col-md-6">
```

Comment:
this caught my eye, I tested with and without ecl-col-12, no difference visually, divs already are 100% width, same thing for the image wrapper below

Reply:
Ok. I've removed this class.